### PR TITLE
docs: move creds before tilt for clarity; fix 404

### PR DESCRIPTION
**Pull request by alexeldeib**
**What this PR does / why we need it**:
Currently if you run through the docs for tilt, you don't ever hit our prerequisites section to generate credentials for Azure. This moves that step from manual testing earlier to clarify it's always necessary.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -146,14 +151,9 @@ make delete-workload-cluster
...
 
-#### Setting up the environment
-
-Your environment must have the Azure credentials as outlined in the [getting
-started prerequisites section](./getting-started.md#Prerequisites)
-
 #### Creating a dev cluster
 
-The steps below are provided in a convenient script in [hack/create-dev-cluster.sh](hack/create-dev-cluster.sh). Be sure to set `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`, `AZURE_SUBSCRIPTION_ID`, and `AZURE_TENANT_ID` before running. Optionally, you can override the different cluster configuration variables. For example, to override the workload cluster name:
+The steps below are provided in a convenient script in [${REPO_ROOT}/hack/create-dev-cluster.sh](../hack/create-dev-cluster.sh). Be sure to set `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`, `AZURE_SUBSCRIPTION_ID`, and `AZURE_TENANT_ID` before running. Optionally, you can override the different cluster configuration variables. For example, to override the workload cluster name:
```
Comment (CecileRobertMichon) [inline docs/development.md:156]: do we need ${REPO_ROOT} in the display name? I thought it was implied (also all our other links in docs don't have it)

Comment (alexeldeib) [reply docs/development.md]: you're right, I was just looking at the bits about repo setup and didn't want to confuse people

Comment (alexeldeib) [reply docs/development.md]: fixed + squashed

**Comment by CecileRobertMichon** [comment]
/lgtm

keep the hold until #516 merges

**Pull request merged**